package.json

```diff
@@ -20,20 +20,23 @@
   "author": "Hristo Oskov",
   "license": "Apache-2.0",
   "dependencies": {
+    "auth0-lock": "^10.3.0",
```

Comment:
`dependencies` are required for the app to run. `devDependencies` are required only for development. anything related to unit tests should be under `devDependencies`. I believe anything that's being bundled will be under `dependencies`.

- react
- react-dom
- react-router (I think)
- react-bootstrap (I think)
- auth0-lock